packages/cozy-pouch-link/src/remote.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Fetch remote instance
+ *
+ * @param {URL} url - The remote instance URL, including the credentials
+ * @param {object} params - The params to query the remote instance
+ * @returns {object} The instance response
+ */
+export const fetchRemoteInstance = async (url, params = {}) => {
+  const username = url.username
+  const password = url.password
+
+  Object.keys(params).forEach(key => {
+    url.searchParams.set(key, params[key])
+  })
+  const fetchUrl = `${url.protocol}//${url.href.substring(
+    url.href.indexOf('@') + 1
+  )}`
+  const headers = new Headers()
+  headers.append('Accept', 'application/json')
+  headers.append('Content-Type', 'application/json')
+  headers.append('Authorization', 'Basic ' + btoa(username + ':' + password))
```

Comment:
@paultranvan can't we use some methods from here: https://github.com/cozy/cozy-client/blob/master/packages/cozy-pouch-link/src/CozyPouchLink.js#L31-L35

Reply:
Sure, f4b7e2f1bac3378f2eee367ec4ed2c4cd2fe3eaf